service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/NotificationServiceTest.java

```diff
@@ -17,15 +20,18 @@
 import static uk.gov.hmcts.reform.fpl.NotifyTemplates.PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
 
 @ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {NotificationService.class, NotificationClient.class, JacksonAutoConfiguration.class})
+@TestPropertySource(properties = {"fpl.env=localhost" })
 public class NotificationServiceTest {
     public static final String TEST_RECIPIENT_EMAIL = "test@example.com";
     public static final String REFERENCE = "12345L";
     public static final String TEMPLATE_ID = PARTY_ADDED_TO_CASE_BY_EMAIL_NOTIFICATION_TEMPLATE;
```

Comment:
can be private, class does not need public

Reply:
done